### PR TITLE
Travis aws

### PR DESCRIPTION
**Pull request by abra7134**
This PR adds kitchen cloud tests in travis.

Also it will be necessary to update .travis/secret_keys.tar.enc, he must include files:
* travis-ci-aws-frank.pem as private key to aws instance
* express42.pem as key for chef supermarket

And .travis.yml also be updated: the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables need to be added as secured in env.global section.

### CODE REVIEW

File: test/fixtures/cookbooks/zabbix_lwrp_test/recipes/zabbix24.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@
 include_recipe 'postgresql_lwrp::apt_official_repository'
 include_recipe 'nginx::official-repo'
 
+# Temporary use higher version of zabbixapi, for correct tests works
+# In gem zabbixapi==2.4.X uses old json gem (==1.6.1) but in chefdk uses newest version
+node.default['zabbix']['api-version'] = '3.0.0'
```
Comment (serjs) [inline test/fixtures/cookbooks/zabbix_lwrp_test/recipes/zabbix24.rb:7]: Are zabbix24.rb and +node.default['zabbix']['api-version'] = '3.0.0' changes inserted as planned?

Comment (abra7134) [reply test/fixtures/cookbooks/zabbix_lwrp_test/recipes/zabbix24.rb:7]: Yes, as planned. It work.
With zabbixapi==2.4.X there are trouble with json gem versions, as described in file's comment.

File: .travis.yml [hunk 2]
```diff
@@ -10,8 +10,9 @@ before_install:
 - tar xf .travis/secret_keys.tar -C .travis
 - mv .travis/travis ~/.ssh/id_rsa
```
Comment (serjs) [inline .travis.yml]: We use .travis/travis-ci-aws-frank.pem in our builds
```transport:
 +  ssh_key: '.travis/travis-ci-aws-frank.pem'
```
in .kitchen.cloud.yml

Can you remove this moving operation?

Comment (abra7134) [reply .travis.yml]: It's removed and I update description.

**Pull request merged**